docs/conf.py

```diff
@@ -0,0 +1,16 @@
+import lablib
+
+project = "LabLib"
+copyright = "2024, maxpareschi, jakubjezek001, doerp"
+author = lablib.__author__
+release = lablib.__version__
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    "revitron_sphinx_theme"
```

Comment:
any particular reason to use this specific rtd fork?
could also go plain rtd🤷

Reply:
i kinda like these two here:
https://pydata-sphinx-theme.readthedocs.io/en/stable/
https://pradyunsg.me/furo/

they come with light/dark mode toggle, nice typo and some colors yay 🙃

Reply:
> any particular reason to use this specific rtd fork?

Not particularly, It had only been used in OpenPype.